Source/WebCore/Modules/fetch/FetchResponse.cpp

```diff
@@ -211,6 +211,8 @@ void FetchResponse::addAbortSteps(Ref<AbortSignal>&& signal)
         if (!weakThis)
             return;
 
+        Ref protectedThis { *this };
```

Comment:
I think our usual pattern when dealing with weak pointers is:
```
RefPtr protectedThis = weakThis.get();
if (!protectedThis)
    return;
```

Reply:
done

Reply:
done